src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/Arbeidsgiverperiode.kt

```diff
@@ -33,4 +48,10 @@ data class RedusertLoennIAgp(
         StreikEllerLockout,
         TidligereVirksomhet,
     }
+
+    internal fun valider(): FeiletValidering? =
+        valider(
+            vilkaar = beloep.erNullEllerOverNullOgUnderMaks(),
```

Comment:
Litt forvirrende (langt) navn - burde kanskje også vært feilmelding MÅ VÆRE STØRRE ELLER LIK NULL, nå leste jeg at beløp større eller lik null er en feil. (og det tror jeg ikke). 
En annen ting er at vi kanskje må tilby skjemaene våre i LPS-API (sikkert som swagger antar jeg.. :/ ) 
Så kanskje validering i klassene blir vanskelig å oversette, at det bør ligge et annet sted. Dette kan jo fint flyttes senere, men la oss tenke på det

Reply:
Feilmelding kan jeg fikse.
Kommer ikke på noe for funksjonsnavnet 🤔🤔🤔 Tanken er at det skal ligne, men også skille seg fra den andre funksjonen `erOverNullOgUnderMaks`, som ikke tillater 0.

Reply:
Kanskje beloep.erStorreEllerLikNullOgUnderMaks() - matcher feilmelding litt bedre også

Reply:
Da bytter jeg til det! ... Og tenker at det er en åpenbar løsning som jeg ikke hadde sjans til å komme på 🙈